File: bestia-models/src/main/java/net/bestia/model/web/AccountRegistration.java
package net.bestia.model.web;

import java.io.Serializable;

import net.bestia.model.domain.Gender;
import net.bestia.model.domain.Hairstyle;

public class AccountRegistration implements Serializable {
	
	private static final long serialVersionUID = 1L;
	
	private String username;
	private String email;
	private String password;
	private Gender gender;
	private Hairstyle hairstyle;
	private String campaignCode;
	private String authToken;

	public AccountRegistration() {
		
	}
}

File: bestia-messages/src/main/kotlin/net/bestia/messages/account/AccountRegistration.kt
package net.bestia.messages.account

/**
 * This POJO holds the data needed for an account registration.
 *
 * @author Thomas Felix
 */
data class AccountRegistration(
    val username: String,
    val email: String,
    val password: String,
    val gender: String,
    val campaignCode: String? = null
)